### ISSUE TITLE
Karma plugin doesn't scan Local files Workaround

### ISSUE THREAD

**Issue by devanshah2** [IBMa/equal-access#4]
##### APPLIES TO:
@ibma/karma plugin

##### PROBLEM DESCRIPTION:
karma plugin  doesn't load the page into iframe and keep trying until run out of tries 

##### STEPS TO REPRODUCE:
(Steps to reproduce the issues from start to finish.)
(Include a sample html file when possible (attach under the "Links" section), Include a link if sample file cannot be produced.)
	1.provide local file to scan
	2.see browser opens and empty iframe 
	3.browser crashes 

##### EXPECTED OUTPUT:
should be able to load the  local file to iframe and scan it 

##### ACTUAL OUTPUT:
doesn't scan local file
